src/BenchmarksServer/Startup.cs

```diff
@@ -194,8 +209,8 @@ private static string CloneAndRestore(string path, ServerJob job)
             }
```

Comment:
I would exclude the benchmarks dir itself from this foreach loop, since we don't need to add `benchmarks\src` to its own global.json.